lib/src/core/client/strategy/strategy.ts

```diff
@@ -70,7 +78,7 @@ const StringOperator = (constraint: Constraint, context: Context) => {
     contextValue = contextValue?.toLocaleLowerCase();
   }
 
-  if(typeof contextValue !== 'string') {
+  if (typeof contextValue !== "string") {
```

Comment:
lots of formatting distractions in this PR? Is there some missing automated check for formatting in our scripts?